src/xkb/compose.rs

```diff
@@ -0,0 +1,95 @@
+use xkb::ffi::*;
+use std::ffi::CString;
+use super::{Context, Keysym};
+
+pub use xkb::ffi::xkb_compose_compile_flags as CompileFlags;
+pub use xkb::ffi::xkb_compose_state_flags as StateFlags;
+
+pub use xkb::ffi::xkb_compose_status as Status;
+pub use xkb::ffi::xkb_compose_feed_result as FeedResult;
+
+
+pub struct Table {
+  ptr: *mut xkb_compose_table,
+}
+
+impl Table {
+  pub fn new(context: &Context, locale: &str, flags: CompileFlags) -> Table {
+    let string = CString::new(locale).unwrap();
+
+    unsafe {
+      Table {
+        ptr: xkb_compose_table_new_from_locale(context.get_raw_ptr(), string.as_ptr(), flags)
+      }
+    }
+  }
+
```

Comment:
The safe wrapper looks quite incomplete. Can you complete it?
e.g: having `Table::new_from_locale`, `Table::new_from_buffer` ...
`Table::new_from_file` would be nice, but I'm not aware of a `FILE` wrapper in Rust. May be it could read the whole file into memory and call `xkb_compose_table_new_from_buffer`.